src/config/validators/index.ts

```diff
@@ -30,14 +30,14 @@ export const VALIDATOR_COMMUNITY = [
     },
   },
   {
-    name: 'Xameyz',
```

Comment:
@bulgakovvlad You have overridden another validator. Please add it next to the other validators.

Reply:
@bulgakovvlad Could you please merge all the changes after this [PR](https://github.com/Cerebellum-Network/staking-dashboard/pull/23) into the dev-cere branch to avoid duplicate work?

Reply:
Please check